src/store/index.ts

```diff
@@ -28,8 +28,9 @@ if (process.env.NODE_ENV !== 'production' && process.env.NODE_ENV !== 'test') {
   middleware.push(loggerMiddleware);
 }
 
+export const localLogger = new Logger();
```

Comment:
I don't think we should have a singleton local logger.  This makes the VegaLite unnecessary coupled.